contrib/camel/samples/README.md

```diff
@@ -16,7 +16,7 @@ These steps assume that you have checked out the repo and have a shell in this
    any namespace where you want to run Camel sources.
 
    The preferred version that is compatible with Camel sources is
-   [Camel K v0.2.0](https://github.com/apache/camel-k/releases/tag/0.2.0).
+   [Camel K v0.3.2](https://github.com/apache/camel-k/releases/tag/0.3.2).
```

Comment:
the PR says 0.3.3 - and that's also here:
https://github.com/knative/eventing-sources/blob/118f0e44a252071efc6f91905cbd8f88e777a16e/Gopkg.toml#L113

Reply:
Yeah, written it before latest release..